Makefile

```diff
@@ -67,14 +67,21 @@ post-install:
 	@# Must remove the .babelrc for 0.42.0 to work correctly
 	@# Need to copy custom ImagePickerModule.java that implements correct permission checks for android
 	@rm node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/ImagePickerModule.java
-	@cp ./ImagePickerModule.java node_modules/react-native-image-picker/android/src/main/java/com/imagepicker
+	@cp ./native_modules/ImagePickerModule.java node_modules/react-native-image-picker/android/src/main/java/com/imagepicker
+	@# Add caching support for native module constants
+	@rm node_modules/react-native-device-info/android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java
+	@rm node_modules/react-native-fetch-blob/android/src/main/java/com/RNFetchBlob/RNFetchBlob.java
+	@cp ./native_modules/RNDeviceModule.java node_modules/react-native-device-info/android/src/main/java/com/learnium/RNDeviceInfo
+	@cp ./native_modules/RNFetchBlob.java node_modules/react-native-fetch-blob/android/src/main/java/com/RNFetchBlob
+
 	@rm -f node_modules/intl/.babelrc
 	@# Hack to get react-intl and its dependencies to work with react-native
 	@# Based off of https://github.com/este/este/blob/master/gulp/native-fix.js
-	@sed -i'' -e 's|"./locale-data/index.js": false|"./locale-data/index.js": "./locale-data/index.js"|g' node_modules/react-intl/package.json
-	@sed -i'' -e 's|"./lib/locales": false|"./lib/locales": "./lib/locales"|g' node_modules/intl-messageformat/package.json
-	@sed -i'' -e 's|"./lib/locales": false|"./lib/locales": "./lib/locales"|g' node_modules/intl-relativeformat/package.json
-	@sed -i'' -e 's|"./locale-data/complete.js": false|"./locale-data/complete.js": "./locale-data/complete.js"|g' node_modules/intl/package.json
+	#@sed -i'' -e 's|"./locale-data/index.js": false|"./locale-data/index.js": "./locale-data/index.js"|g' node_modules/react-intl/package.json
```

Comment:
for this perhaps we can keep those that we are currently supporting instead of removing them all but es-US ?

Reply:
This was a test to show that reducing the JS bundle size helps in compiling when the app starts.  Like stated in this PR's description. this is not a good change and will require another way to avoid increasing the JS bundle size and also support all localization files.

A thought I had was to support it via a native module. I read it may not be necessary you may be able to use a native resource file via `require`.  At least it works for Images.  Crossing fingers that it works with json files in a packaged apk 

https://medium.com/@chrissullivan.dev/resource-handling-in-react-native-3e5b4a2c84ec

Although, the more I read it seems like it still bundles that.  I'll continue looking but one things for sure.  Including the localization files increases the bundle size by double